### PR TITLE
Remove excess files and gems

### PR DESCRIPTION
**Pull request by kienan**
What: Cleans up the gem file to remove gems we don't use anymore and removes files like the word cloud and doge stuff.

Why: We want our gemfile to be clean so it's easier to explain to new members and also easier for us to read.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -96,26 +92,16 @@ group :development, :test do
 end
 
 gem 'nprogress-rails'
-# Use unicorn as the app server
-# gem 'unicorn'
 
 # Use Capistrano for deployment
```
Comment (kienan) [inline Gemfile]: Can we remove this as well?

File: app/views/users/settings.html.haml [hunk 2]
```diff
@@ -1,35 +0,0 @@
...
-    else
-    {
-      $("#word-cloud-switch").bootstrapSwitch('state', true, true);
-
-      if (doge)
-      {
-        $("#doge-switch").bootstrapSwitch('state', true, true);
-      }
-    }
-  });
```
Comment (atallahhezbor) [inline app/views/users/settings.html.haml]: The devise views use haml as well. Worth converting?

File: app/views/users/settings.html.slim [hunk 3]
```diff
@@ -2,36 +2,4 @@
...
-    {
-      $("#word-cloud-switch").bootstrapSwitch('state', true, true);
-
-      if (doge)
-      {
-        $("#doge-switch").bootstrapSwitch('state', true, true);
-      }
-    }
-  });
+  | Settings
```
Comment (atallahhezbor) [inline app/views/users/settings.html.slim]: Should we even have this page anymore? There is currently no route to it

File: app/views/users/settings.html.slim [hunk 4]
```diff
@@ -2,36 +2,4 @@
...
-    else
-    {
-      $("#word-cloud-switch").bootstrapSwitch('state', true, true);
-
-      if (doge)
-      {
-        $("#doge-switch").bootstrapSwitch('state', true, true);
-      }
-    }
-  });
```
Comment (atallahhezbor) [inline app/views/users/settings.html.slim]: Should probably also remove the controller methods for this in the users controller

File: app/views/users/settings.html.slim (hunk 3 above)
Comment (kienan) [inline app/views/users/settings.html.slim]: Good catch -- deleted!

File: app/views/users/settings.html.haml (hunk 2 above)
Comment (kienan) [inline app/views/users/settings.html.haml]: I think we should save this for another PR.

File: app/views/users/settings.html.slim (hunk 4 above)
Comment (kienan) [inline app/views/users/settings.html.slim]: Truu -- deleted the users controller.

File: Gemfile (hunk 1 above)
Comment (atallahhezbor) [inline Gemfile]: I believe so but I don't know our gem usage that well. Alan? @aw3as

Comment (kienan) [inline Gemfile]: I decided to keep it in bc we should look into using it for deployment soon.

**Pull request merged**